projects/dsp-ui/src/lib/action/components/select-project/select-project.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="dsp-select-project-component">
+    <form class="form-content">
+        <mat-form-field class="large-field select-project">
+
+            <input matInput [matAutocomplete]="project" [formControl]="selectProjectForm.controls['projectName']"
```

Comment:
HTML code could be neat aligned as mentioned in [coding convention](https://docs.google.com/document/d/1SIVMxoV0N8xYqhVk_0hhuqSuUF47D1YYrDESqdmipfM/edit#heading=h.cm3zjzy3j1vx). To do so you should install recommended **Prettier** extension and align it manually or enable alignment on save in **vscode** settings.

Reply:
@mpro7 Good point. I use the default "Format Document" or "Format Selection" command in vsc. We had some issues with "Prettier" especially in html files.

Reply:
@kilchenmann what issues exactly? I personally resigned from Prettier, since devs didn't want to extend formatting to some settings. **Format Document/Selection** depends on the default formatter and defined rules. I think we could revise the coding/formatting convention and provide the best possible solution to automate the rules, as well as get settings aligned.

Reply:
@mdelez mentioned some issues with prettier. I do not remember which ones exactly. And I thought @waychal is using anyway webstorm and not vsc. We should discuss the tools to format code in another task (maybe) on youtrack...

Reply:
@kilchenmann @mpro7 Yes, I use default formatting from webstorm. I will check the conventions in the link provided above by Marcin.

Reply:
I've disabled Prettier. I think we were using the default config file and it was not formatting it the way we wanted. Maybe if we spend some more time on the config file, we can get it to act how we want it to.

Reply:
Discussion has been moved to DSP-936

Reply:
Done